sfdx: Add page (#9255)
tldr-pages/tldr: pages/common/sfdx.md

# sfdx

> Salesforce CLI is a powerful command line interface that simplifies development and build automation when working with your Salesforce org.
> More information: <https://developer.salesforce.com/tools/sfdxcli>.

- Authorize a Salesforce Org:

> `sfdx force:auth:web:login -a {{Org Alias}} -r {{Org URL}}`

Reviewer: ```suggestion
`sfdx force:auth:web:login -a {{org_alias}} -r {{org_URL}}`
```

> `sfdx force:auth:web:login -a {{org_alias}} -r {{org_url}}`

Reviewer 2: We try to avoid abbreviations
```suggestion
`sfdx force:auth:web:login -a {{organization}} -r {{organization_url}}`
```

> - List all authorized Orgs:

Reviewer 2: ```suggestion
- List all authorized organizations:
```

`sfdx force:org:list`

> - Open an Org in your Browser:

Reviewer 2: ```suggestion
- Open a specific organization in the default web browser:
```

> `sfdx force:org:open -u {{Org Alias}}`

Reviewer: ```suggestion
`sfdx force:org:open -u {{org_alias}}`
```

> `sfdx force:org:open -u {{org_alias}}`

Reviewer 2: We prefer long arguments if available
```suggestion
`sfdx force:org:open --targetusername {{organization}}`
```

> - Display Org information:

Reviewer 2: ```suggestion
- Display information about a specific organization:
```

> `sfdx force:org:display -u {{Org Alias}}`

Reviewer: ```suggestion
`sfdx force:org:display -u {{org_alias}}`
```

> `sfdx force:org:display -u {{org_alias}}`

Reviewer 2: ```suggestion
`sfdx force:org:display --targetusername {{organization}}`
```

- Push source meta data to an Org:

> `sfdx force:source:push -u {{Org Alias}}`

Reviewer: ```suggestion
`sfdx force:source:push -u {{org_alias}}`
```

> - Push source meta data to an Organization:

Reviewer 2: ```suggestion
- Push source metadata to an Organization:
```

- Pull source meta data from an Org:

> `sfdx force:source:pull -u {{Org Alias}}`

Reviewer: ```suggestion
`sfdx force:source:pull -u {{org_alias}}`
```

> `sfdx force:source:pull -u {{org_alias}}`

Reviewer 2: ```suggestion
`sfdx force:source:pull --targetusername {{organization}}`
```

> - Generate a password for your Org user:

Reviewer 2: ```suggestion
- Generate a password for the organization logged in user:
```

> - Pull source meta data from an Organization:

Reviewer 2: ```suggestion
- Pull source metadata from an Organization:
```

> `sfdx force:user:password:generate -u {{Org Alias}}`

Reviewer: ```suggestion
`sfdx force:user:password:generate -u {{org_alias}}`
```

> `sfdx force:user:password:generate -u {{org_alias}}`

Reviewer 2: ```suggestion
`sfdx force:user:password:generate --targetusername {{organization}}`
```

> - Assign a permission set to your Org user:

Reviewer 2: ```suggestion
- Assign a permission set to for the organization logged in user:
```

> - Generate a password for the organization logged in user:

Reviewer: ```suggestion
- Generate a password for the organization's logged-in user::
```
I think this would be better.

> - Generate a password for the organization's logged-in user::

Reviewer: ```suggestion
- Generate a password for the organization's logged-in user:
```

> `sfdx force:user:permset:assign -n {{Permission Set Name}} -u {{Org Alias}}`

Reviewer: ```suggestion
`sfdx force:user:permset:assign -n {{permission_set_name}} -u {{org_alias}}`
```

> `sfdx force:user:permset:assign -n {{permission_set_name}} -u {{org_alias}}`

Reviewer 2: ```suggestion
`sfdx force:user:permset:assign --permsetname {{name}} --targetusername {{organization}}`
```

> - Assign a permission set for the organization logged in user:

Reviewer: ```suggestion
- Assign a permission set for the organization's logged-in user:
```
